pkgs/top-level/perl-packages.nix

```
@@ -16737,7 +16755,7 @@ let
     };
     buildInputs = [ LWP ModuleBuildTiny TestRequires TestTCP ];
     nativeBuildInputs = stdenv.lib.optional stdenv.isDarwin shortenPerlShebang;
-    propagatedBuildInputs = [ DataDump HTTPParserXS NetServer Plack ];
+    propagatedBuildInputs = [ DataDump HTTPParserXS NetServer Plack NetServerSSPrefork ];
```

```suggestion
    propagatedBuildInputs = [ DataDump HTTPParserXS NetServer Plack ];
```

Is adding this to `propagatedBuildInputs` for Starman needed?

---

Yes; see the PR description. This is an optional dependency of Starman which previously wasn't included, and if you tried to use the feature that this is used for, Starman would crash.